### PR TITLE
Repositories

### PR DESCRIPTION
**Pull request by MaksymVakuliuk**

### CODE REVIEW

File: src/main/java/com/spring/boot/project/demo/servise/ReviewsToDbPutter.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class ReviewsToDbPutter implements DbPutter {
+    private static final String REVIEW_CSV_FILE = "src/main/resources/csv/Reviews.csv";
```
Comment (RomanDubovskyi) [inline src/main/java/com/spring/boot/project/demo/servise/ReviewsToDbPutter.java]: it's hardcode, make your method receive path as input parameter

File: src/main/java/com/spring/boot/project/demo/servise/ReviewsToDbPutter.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    private final AmazonUserRepository amazonUserRepository;
+    private final ProductRepository productRepository;
+    private final ReviewRepository reviewRepository;
+    private final Parser<Review> parser;
+    private final FileReader fileReader;
+
+    @Override
+    public void putDataToDatabase() {
```
Comment (RomanDubovskyi) [inline src/main/java/com/spring/boot/project/demo/servise/ReviewsToDbPutter.java]: try not to use void methods, it's impossible to test them

File: src/main/java/com/spring/boot/project/demo/servise/DbPutter.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package com.spring.boot.project.demo.servise;
+
+public interface DbPutter {
```
Comment (RomanDubovskyi) [inline src/main/java/com/spring/boot/project/demo/servise/DbPutter.java]:
```suggestion
public interface DbInitializer {
```

File: src/main/java/com/spring/boot/project/demo/servise/ReviewsToDbPutter.java (hunk 2 above)
Comment (MaksymVakuliuk) [reply src/main/java/com/spring/boot/project/demo/servise/ReviewsToDbPutter.java]: What can we return to test them? If we use boolean, tests wouldn't be objective.

Comment (RomanDubovskyi) [reply src/main/java/com/spring/boot/project/demo/servise/ReviewsToDbPutter.java]: yep, you're right, in this case you can leave it with void....

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**